### PR TITLE
Add terminator route overrides

### PR DESCRIPTION
**Pull request by max-lobur**
Tested on dev:
https://app.clubhouse.io/vgs/story/8674/terminators-isolate-customers-to-specific-eb-stacks#activity-9239

### CODE REVIEW

**Comment by max-lobur** [comment]
[ch8674]

**Comment by max-lobur** [comment]
cc @akhon

File: defaults/main.yml [hunk 1]
```diff
@@ -11,3 +11,5 @@ dns_resolver: "{{ ansible_dns.nameservers | join (',') }}"
 
 cert_mgr_url: http://certificates.verygoodsecurity.io
 ssl_router_tld: verygoodproxy.io
+ssl_router_upstream_overrides_tnt: {}
```
Comment (zdmytriv) [inline defaults/main.yml]: I would add comment with example of how this dictionary should be populated.
```
ssl_router_upstream_overrides_tnt: {
    "tnt12345678", "reverse.jail.verygoodproxy.com"
}
```

File: defaults/main.yml [hunk 2]
```diff
@@ -11,3 +11,5 @@ dns_resolver: "{{ ansible_dns.nameservers | join (',') }}"
...
+ssl_router_upstream_overrides_chn: {}
```
Comment (zdmytriv) [inline defaults/main.yml]: Here as well

Comment (zdmytriv) [reply defaults/main.yml]:
```
ssl_router_upstream_overrides_chn: {"customer.live.verygoodproxy.com", "reverse.jail.verygoodproxy.com"}
```

File: defaults/main.yml (hunk 1 above)
Comment (max-lobur) [reply defaults/main.yml]: https://github.com/verygood-ops/infra-vault/pull/1725

File: defaults/main.yml (hunk 2 above)
Comment (max-lobur) [reply defaults/main.yml]: in dev https://github.com/verygood-ops/infra-vault/pull/1725
```
ssl_router_upstream_overrides_tnt:
  tnt8ciq1xog: reverse-iso1.sandbox.verygoodproxy.io.
ssl_router_upstream_overrides_chn:
  tenant1a.testclients.verygoodsecurity.io: reverse-iso1.sandbox.verygoodproxy.io.
```

Comment (zdmytriv) [reply defaults/main.yml]: 👍

**Comment by akhon** [comment]
@max-lobur there was a change for LU environment on catch-all config. could you please rebase on latest master and make changes over it?

**Comment by max-lobur** [comment]
@akhon rebased

**Pull request merged**